scripts/backup.sh

```diff
@@ -179,3 +179,17 @@ backup_klipperscreen(){
     ERROR_MSG=" Can't backup KlipperScreen directory!\n Not found!"
   fi
 }
+
+backup_MoonrakerTelegramBot){
```

Comment:
it must be called `backup_MoonrakerTelegramBot(){`

Reply:
ooops

Reply:
oops x2 :)